### DISCUSSION TITLE
HELP ME, I am getting this CompileError from past few days and all the methods I've tried haven't worked

### DISCUSSION THREAD

**Discussion by Abhijeet1005** [smartcontractkit/full-blockchain-solidity-course-py#1537] [General]
![2022-06-12](https://user-images.githubusercontent.com/46476792/173231864-6ffd8042-b237-4e06-a866-72a1a5e0fda6.png)

I was simply doing the tutorial but at one point this error started, I tried changing the solidity version in the settings but the error is still there and I also tried after changing the remote version but nothing works, HELP ME in need of some

**Comment by cromewar** [comment]
Hello @abhijeet10052002 
Please change the compiler to local, and then back to remote, and also select 0.6.6.
I hope this might help.
